src/com/android/calendar/month/MonthWeekEventsView.java

```diff
@@ -159,6 +162,18 @@
     private ObjectAnimator mTodayAnimator = null;
     private int[] mDayXs;
 
+    private static int EVENT_RECT_ROUNDING = 5;
+    private static int EVENT_RECT_ALLDAY_ROUNDING = 5;
+    private static int EVENT_RECT_MARGIN_LEFT = 5;
+    private static int EVENT_RECT_MARGIN_RIGHT = 5;
+    private static int EVENT_RECT_MARGIN_TOP = 5;
+    private static int EVENT_RECT_MARGIN_BOTTOM = 5;
+
+    private static int EVENT_RECT_TEXT_MARGIN_LEFT = 5;
+    private static int EVENT_RECT_TEXT_MARGIN_RIGHT = 5;
+    private static int EVENT_RECT_TEXT_MARGIN_TOP= 0;
```

Comment:
why 0?

Reply:
Because if it has an greater offset, the text moves out of the drawingboundaries for the colored chip behind it. I put it in because of completeness, but it could be removed